### ISSUE TITLE
Post login redirect functionality

### ISSUE THREAD

**Issue by DaveOrDead** [luukhaijes/kinde-angular#12]
Hey @luukhaijes - Dave here from Kinde. Again, thanks for your awesome work on this :) 

We've had a request come in to add `post_login_redirect_url` similar to the functionality available in the Kinde NextJS SDK.

This is the code they were hoping to use.
```
export class KindeAuthProviderGuard extends AuthProviderGuard {

    protected redirectIfUnauthenticated(state: RouterStateSnapshot): Observable<boolean> {
        const authService = inject(KindeAngularService);
        return authService.isAuthenticated$.pipe(
            take(1),
            tap(async isAuthenticated => {
                if (!isAuthenticated) {
                    await authService.login({
                        post_login_redirect_url: state.url
                    });
                }
            })
        );
    }
}
```

Unfortunately my Angular knowledge isn't there to implement this myself

**Comment by luukhaijes** [comment]
Hi Dave, thanks for opening this issue. As I can see I don't have a handler after login which does the redirection to the value given in `post_login_redirect_url`. I will create a fix for this.

**Comment by luukhaijes** [comment]
Fixed for a while by #13
